shellexporter/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM alpine:3.14
+RUN apk add --no-cache go git jq bash make
+RUN mkdir /usr/local/shellexporter
+
+workdir /usr/local/shellexporter
+
+RUN git clone https://github.com/dodopizza/prometheus-shell-exporter.git  ./
+RUN sleep 60
```

Comment:
Not smart to be honest but it works. However after the image download the go command triggers another download of different GO-modules that are needed. Without the sleep I had problems running the docker build successfully. The process got on without waiting for the GO-modules for beeing downloaded and due to that afterwards it crashed when it was trying to start.
Any suggestions for exchange the time based to a conditional wait?

Reply:
I removed the "sleeps" because I could reproduce the problems with dependent package installation. Even with cleaned repos and caches building the images from scratch.
Please test and take care of go-modules-dependencies.